### ISSUE TITLE
class conditional batch norm not implemented

### ISSUE THREAD

**Issue by djl11** [heykeetae/Self-Attention-GAN#33]
Just want to verify that I am correct that the class condition batch norm is not implemented in this repo?
In the original tensorflow [code](https://github.com/brain-research/self-attention-gan/blob/master/generator.py), the network blocks incorporate a ConditionalBatchNorm operation.

I would just like a simple answer if my conclusion is correct, as this could help others asking themselves the same question :)
